website/src/pages/showcase.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import clsx from 'clsx';
+import Layout from '@theme/Layout';
+import Link from '@docusaurus/Link';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import styles from '../pages/showcase.module.css';
```

Comment:
Why not just `./showcase.module.css`? Why go ../pages?

Reply:
Good point! updated.